src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,16 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null || user.getPassword().length() < 6
+                || user.getAge() < 18 || user.getLogin() == null) {
+            throw new RuntimeException();
+        }
```

Comment:
split this if on 4 and throw exceptions with informative messages

Reply:
Ok, thanks. I understand